src/nls/en/strings.json

```diff
@@ -94,7 +94,11 @@
                 "HEADLINE": "Design Space",
                 "BODY_FIRST": "Twitter",
                 "BODY_SECOND": "GitHub",
-                "BODY_THIRD": "Forum"
+                "BODY_THIRD": "Forum",
+                "BODY_ADVERTISEMENT_1": "You seem like the type who likes to check out betas. We've been hard at work building a new end-to-end solution for UX designers, including great integration with Photoshop. We're happy to share that",
+                "BODY_ADVERTISEMENT_2": "Adobe Experience Design CC",
+                "BODY_ADVERTISEMENT_3": "is now in public beta.",
```

Comment:
Oh, after it is committed to master branch.

From: Shaoshing <notifications@github.com<mailto:notifications@github.com>>
Reply-To: adobe-photoshop/spaces-design <reply@reply.github.com<mailto:reply@reply.github.com>>
Date: Tuesday, April 19, 2016 at 3:36 PM
To: adobe-photoshop/spaces-design <spaces-design@noreply.github.com<mailto:spaces-design@noreply.github.com>>
Cc: fzhu <fzhu@adobe.com<mailto:fzhu@adobe.com>>, Mention <mention@noreply.github.com<mailto:mention@noreply.github.com>>
Subject: Re: [adobe-photoshop/spaces-design] Add a Mac-only ad for Adobe Xd in the first-launch dialog (#3810)


In src/nls/en/strings.json<https://github.com/adobe-photoshop/spaces-design/pull/3810#discussion_r60324318>:

> @@ -94,7 +94,11 @@
>                  "HEADLINE": "Design Space",
>                  "BODY_FIRST": "Twitter",
>                  "BODY_SECOND": "GitHub",
> -                "BODY_THIRD": "Forum"
> +                "BODY_THIRD": "Forum",
> +                "BODY_ADVERTISEMENT_1": "You seem like the type who likes to check out betas. We've been hard at work building a new end-to-end solution for UX designers, including great integration with Photoshop. We're happy to share that",
> +                "BODY_ADVERTISEMENT_2": "Adobe Experience Design CC",
> +                "BODY_ADVERTISEMENT_3": "is now in public beta.",


Sorry for not being clear. React can render an array of mixed objects. I think we can make use of that by passing in an arrary that contains pure string and wrapped string:

var description = nls.localize("strings.FIRST_LAUNCH.SLIDES.7.BODY_ADVERTISEMENT").split("{ADOBE_XD_CC}"),
    name = (<em>nls.localize("strings.FIRST_LAUNCH.SLIDES.7.ADOBE_XD_CC")</em>);

description.splice(1, 0, name); // insert the namereturn (
  <div>
    {description}
  </div>
);

—
You are receiving this because you were mentioned.
Reply to this email directly or view it on GitHub<https://github.com/adobe-photoshop/spaces-design/pull/3810/files/dc63baae12b5842d16ed6653cbbd81c53c182572#r60324318>